dd-trace-api/src/main/java/datadog/trace/api/Config.java

```diff
@@ -120,6 +120,9 @@
   public static final String MAX_SPANS_PER_TRACE = "max.spans.per.trace";
   public static final Integer DEFAULT_MAX_SPANS_PER_TRACE = 0;
 
+  public static final String MAX_CONTINUATION_DEPTH = "max.continutation.depth";
+  public static final Integer DEFAULT_MAX_CONTINUATION_DEPTH = 100;
```

Comment:
What's the confidence level with this default?  Could a truly runaway trace warrant a greater one to distinguish from complex middleware?